src/ILVerify/src/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("ILVerify.Tests")]
```

Comment:
The ILImporter (and everything from the CoreRT type system under Common) should stay internal. There should be a public façade for the few pieces that need to be public.

I am fine with creating an issue about defining what we want to be public, and switching tests to run against this surface later.